deps/fdasrsf/DynamicProgrammingQ2.c

```diff
@@ -0,0 +1,37 @@
+#include <math.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "dp_grid.h"
+
+void DynamicProgrammingQ2(double* Q1, double* T1, double* Q2, double* T2,
+                          int m1, int n1, int n2,
+                          double* tv1, double* tv2, int n1v, int n2v, double* G,
+                          double* T, double* size, double lam1)
+{
+    int* idxv1;
+    int* idxv2;
+    double* E; /* E[ntv1*j+i] = cost of best path to (tv1[i],tv2[j]) */
+    int* P; /* P[ntv1*j+i] = predecessor of (tv1[i],tv2[j]) along best path */
+
+    idxv1 = malloc((n1v)*sizeof(int));
```

Comment:
Please remove the unnecessary parentheses, and put spaces around the operators. I also prefer `sizeof(*idxv1)` instead of `sizeof(int)` to guarantee that the size is always ok.

Reply:
Done in de19847.